specification/2.0/README.md

```diff
@@ -251,7 +251,7 @@ For property names, glTF uses [camel case](http://en.wikipedia.org/wiki/CamelCas
 
 ## Coordinate System and Units
 
-glTF uses a right-handed coordinate system, that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up. The front of a glTF asset faces +Z.
+glTF uses a right-handed coordinate system; that is, the cross product of right and forward yields up. glTF defines +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
```

Comment:
```suggestion
glTF uses a right-handed coordinate system; that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
```

Reply:
Arguable, the whole "cross product" part is redundant as it's true for any right-handed system.